lib/mumuki/bibliotheca/sinatra/guides.rb

```diff
@@ -46,10 +49,12 @@ def slice_guide_resource_h_for_api(guide)
   end
 
   post '/guides/import/:organization/:repository' do
+    authorize! :writer
```

Comment:
This line may break Github post commit hooks

Reply:
ah, yeah. That slipped my mind!

I’d like to protect this endpoint somehow though 🤔

Reply:
We just discussed that although it _would_ be possible, it would require a lot of work in content, and the impact is not really critical either way.